test/inductor/test_torchinductor.py

```diff
@@ -1945,12 +1945,13 @@ def forward(self, x):
                 res = self.linear(res)
                 return res
 
-        with torch.no_grad():
-            m = M(224, 224).bfloat16().eval()
-            m_opt = torch.compile(m)
-            x = torch.randn(224, 224, dtype=torch.bfloat16)
-            m_opt(x)
-            self.assertEqual(m(x), m_opt(x))
+        if has_bf16_support():
+            with torch.no_grad():
+                m = M(224, 224).bfloat16().eval()
+                m_opt = torch.compile(m)
+                x = torch.randn(224, 224, dtype=torch.bfloat16)
+                m_opt(x)
+                self.assertEqual(m(x), m_opt(x))
```

Comment:
Nit: is there a way to check if the weight change did happen?

Reply:
I guess we have to let users to specify the weights are constant from the frontend, something like `torch.jit.freeze`.

Reply:
Yes, Let's do it at the next step.